### PR TITLE
Feature: assos cemetery

### PR DESCRIPTION
**Pull request by MercierCorentin**
This is a proposition for a first version of the asso's cemetery :kissing_heart: 
#132 resolution.
Makes #283 less common 
Must be pushed into production really carefully

Feature behavior:
- To put an association into the cemetery, set a date for the `in_cemetery_at` field in the administrator page
- Dead assos are displayed in '/assos' when a checkbox is active
- A dead asso's name is decorated with a line-through
- Introduces dead assos pages
- In the associative career, dead associations are displayed the same way as in the first list.

Some screenshots:
image
image
image

### CODE REVIEW

File: app/Console/Commands/Test.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function handle()
         $this->info(PHP_EOL);
         $this->info(PHP_EOL);
 
-        $this->output->success('Code parfait √');
+        $this->output->success('Perfect code √');
```
Comment (cesar-richard) [inline app/Console/Commands/Test.php]: Mouai le wording ^^Plutot Code Valide :)

File: database/seeds/AssosTableSeeder.php [hunk 2]
```diff
@@ -35,6 +35,7 @@ public function run()
 Viens, on est bien !
 DESC
         ,
+                'in_cemetery_at' => null,
```
Comment (cesar-richard) [inline database/seeds/AssosTableSeeder.php]: Bof, pourquoi la valeur n'est pas par defaut a null ? la creation se chargerait de faire tout ca en implicit

File: resources/assets/react/components/AssoCard.js [hunk 3]
```diff
@@ -11,7 +11,7 @@
 import React from 'react';
 import bdeImage from '../../images/bde.jpg';
 
-const AssoCard = ({ image, login, name, shortname, additionalInfo }) => {
+const AssoCard = ({ image, login, name, shortname, additionalInfo, deleted }) => {
```
Comment (cesar-richard) [inline resources/assets/react/components/AssoCard.js]: deleted ou inCemetery ?

**Review by cesar-richard — CHANGES_REQUESTED** [review]

File: database/migrations/2019_11_22_143951_add_cemetary_assos_table.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (cesar-richard) [inline database/migrations/2019_11_22_143951_add_cemetary_assos_table.php]: Et donc ca, tu l'as déjà extrait dans une PR a part ?

**Review by cesar-richard — CHANGES_REQUESTED** [review]

File: database/migrations/2019_11_22_143951_add_cemetary_assos_table.php (hunk 4 above)
Comment (MercierCorentin) [reply database/migrations/2019_11_22_143951_add_cemetary_assos_table.php]: Fait!

**Pull request merged**